### PR TITLE
Ml production

### LINKED ISSUE #27: ML model

### PR DESCRIPTION
**Pull request by bhroben**
Closes #27

### CODE REVIEW

**Comment by bhroben** [comment]
@diegobonato 
some suggestions:
- always open a MR with the branch you are working with, because we can have discussions here and it is easier to check what you are working one. also the pipeline get triggered every time you push something
- python scripts (in models) should be in the python package (policyML)
- write functions or classes instead of scripts because it is easier to integrate them inside the code

File: models/rf_train.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# drop the ID
+data.drop(columns=["id",'dwh_create_data'],inplace=True)
+
+
+# Train test split
+X = data.drop(columns="amt_claim")
+y = data["amt_claim"]
+
+X_train, X_test, y_train, y_test = train_test_split(X,y,test_size=.2)
```
Comment (bhroben) [inline models/rf_train.py:31]: here a random seed was not set so the model saved in models folder can't be compared to the ones in the notebook "model_experiments"

**Comment by bhroben** [comment]
Main changes:
- notebooks are now compiled on mkdocs. To do this i had to set a hard symlink between the notebook folder and the docs/docs/notebook folder. soft symlinks were not found by mkdocs. hard symlinks for github looks just a copy of the file(?)
- tested some models in model_experiments notebooks. compared with an heuristic model that just output the mean value. RMSE went from 95M to 2.2M for the best model ( classifier + regression)

TODO:
- [ ] python scripts need refactoring into functions
- [ ] select which model to use for production and save weights
